split-sequence.asd

```diff
@@ -8,7 +8,12 @@
   :license "MIT"
   :version (:read-file-form "version.sexp")
   :components ((:static-file "version.sexp")
-               (:file "split-sequence")))
+               (:file "package")
+               (:file "vector")
+               (:file "list")
+               #+(or sbcl abcl) (:file "extended-sequence")
```

Comment:
It's better to use ASDF feature expressions here. `(:file "extended-sequence" :if-feature (:or :sbcl :abcl))`. That way the system definition is aware of the feature flags instead of being oblivious to it.

Reply:
Fixed in 08b109a